molecule_libvirt/playbooks/create.yml

```diff
@@ -0,0 +1,259 @@
+---
+
+- name: Create
+  hosts: localhost
+  gather_facts: false
+  no_log: "{{ molecule_no_log }}"
+  vars:
+    molecule_scenario_name: "{{ lookup('env', 'MOLECULE_SCENARIO_NAME') }}"
+    user_home_directory: "{{ lookup('env', 'HOME') }}"
+    molecule_dns_suffix: "molecule.lab"
+    molecule_network_name: "molecule-libvirt"
+    molecule_ssh_user: "molecule"
+    molecule_private_key_path: "{{ lookup('env', 'MOLECULE_EPHEMERAL_DIRECTORY') }}/id_ssh_rsa"
+    libvirt_default_image_path: "{{ user_home_directory}}/.local/molecule/{{ molecule_scenario_name }}/vms"
+    libvirt_vm_arch: "x86_64"
+    libvirt_uri: "qemu:///system"
+    default_image_url: "https://mirror.csclub.uwaterloo.ca/fedora/linux/releases/33/Cloud/x86_64/images/Fedora-Cloud-Base-33-1.2.x86_64.qcow2"
+    default_disk_size: "15G"
+    default_memory_size: "1"
+    default_vcpu: "1"
+    default_ssh_port: "22"
+    mac: "52:54:00:ab:cd:"  # first 5 digits
```

Comment:
The comment should be: "first 5 two-digit hexadecimal numbers"
This could avoid some confusion.